src/plugin/plugin.ts

```diff
@@ -1487,6 +1511,99 @@ class Purchases {
     );
   }
 
+  /**
+   * iOS 15+ only. Presents a refund request sheet in the current window scene for
+   * the latest transaction associated with the active entitlement.
+   *
+   * If the request was unsuccessful, no active entitlements could be found for
+   * the user, or multiple active entitlements were found for the user,
+   * the promise will return an error.
+   * If called in an unsupported platform (Android or iOS < 15), an `UnsupportedPlatformException` will be thrown.
```

Comment:
I think right now this will success if called on Android? Also, this name might not apply on iOS.

Reply:
right, I copied this from React Native, I will update